src/canvas/SelectableCanvas.ts

```diff
@@ -667,10 +667,23 @@ export class SelectableCanvas<
    * @return {Boolean}
    */
   isTargetTransparent(target: FabricObject, x: number, y: number): boolean {
+    const ctx = this.contextCache;
+    const retina = this.getRetinaScaling();
+    const size = Math.max(2 * this.targetFindTolerance * retina, 10);
```

Comment:
Why would you max it out?

Reply:
i would rather have a method `setTargetFindTolerance` that creates this mini canvas, i m not even sure retina scaling is a requirement since the mouse pixels ( the so called css pixels ) are always without retina.

Reply:
> i would rather have a method `setTargetFindTolerance` that creates this mini canvas,

will do

I wasn't sure about retina. What will happen in case of ay scaled and blurry image? Will the hit test return a false positive?

Reply:
I thought I removed the max value of 10